libs/designsystem/src/lib/scss/base/_lists.scss

```
@@ -0,0 +1,89 @@
+$bullet-font-size: 1.3em;
+$hollow-bullet-font-size: 0.8em;
```

Rename Sass variables, e.g.:
```suggestion
$marker-disc-font-size: 1.3em;
$marker-cirle-font-size: 0.8em;
```